lib/PipelineC/PipelineC.cpp

```diff
@@ -30,6 +31,35 @@
 using namespace pipeline;
 using namespace ::revng::pipes;
 
+template<typename T>
+concept default_constructible = std::is_default_constructible<T>::value;
+
+template<default_constructible T>
+class Wrapper {
```

Comment:
Not a great name. `ExistingOrTemporary`? `ExistingOrNew`?

Reply:
`ExistingOrDefault`?